content/general/concepts/what-is-UUID/what-is-UUID.md

```diff
@@ -0,0 +1,32 @@
+---
+Title: 'What is UUID?'
+Description: 'UUID is a universally unique identifier. It can be generated on any computer without the use of a centralized authority. It's a string of 128 bits that is supposed to be unique with little to no chance of clashing with any other UUID produced by anybody else in the world. UUIDs are usually used in database tables as unique keys. They are useful any time you need an ID that needs to be unique in an application or system.'
+Subjects:
+	- 'Computer Science'
+  - 'Web Development'
+Tags:
+	- 'Computer Science'
+  - 'Web Development'
+CatalogContent:
+  - 'paths/front-end-engineer-career-path'
+  - 'paths/computer-science'
+---
+
+UUID is a universally unique identifier. It can be generated on any computer without the use of a centralized authority.
+It's a string of 128 bits that is supposed to be unique with little to no chance of clashing with any other UUID produced by anybody else in the world.
+
+UUIDs are usually used in database tables as unique keys.
+They are useful any time you need an ID that needs to be unique in an application or system.
+
+## UUID example in JavaScript
+
+```js
+const { v4: uuidv4 } = require('uuid');
+
+console.log(uuidv4());
+
+// outputs: 7637c099-79c8-4685-ad47-b1b985f7ac2b
+```
+
+This example shows how to generate a UUID with the `uuidv4()` function.
+This is using the npm package `uuid` and you can install it with `npm i uuid`.
```

Comment:
This sentence should actually be moved up above the JS example so that readers have a quick reference for what command is needed to install. 

Also, let's rename `npm i uuid` to `npm install uuid` (for any readers not familiar with npm-shorthand) and move to it's own line, like so: 

UUIDs can be used in JavaScript with the `uuid` package and installed via npm: 

```bash
npm install uuid
```